packages/components/.eslintrc.js

```diff
@@ -18,6 +18,7 @@ module.exports = {
   ],
   env: {
     browser: true,
+    node: true,
```

Comment:
I have renamed the codemod files (adding a `.txt` to the extension) so you can remove this now

Reply:
I removed this at the rebase